src/Form/DeleteExpressionForm.php

```diff
@@ -10,7 +10,7 @@
 use Drupal\Core\Form\ConfirmFormBase;
 use Drupal\Core\Form\FormStateInterface;
 use Drupal\rules\Entity\ReactionRuleConfig;
-use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Drupal\rules\Exception\NotFoundHttpException;
```

Comment:
I think we should not touch the exceptions that are already fully namespaced. This patch should only replace the global exceptions that don't have a namespace.